static/config/status-chain-genesis.json

```diff
@@ -6,10 +6,10 @@
      "65C01586aa0Ce152835c788aCe665e91Ab3527b8": {
       "balance": "0x1B1AE4D6E2EF500000"
     },
-     "F35E0325dad87e2661c4eF951d58727e6d583d5c": {
+     "bF164ca341326a03b547c05B343b2E21eFAe24b9": {
```

Comment:
I wonder if two accounts above are needed anymore...?

Reply:
It looks like the 1st account is used in `rpc_test.go`, but just to check that `eth_getBalance` call returns > 0. This could be replaced w/ 1 of the new accounts.

The 2nd account doesn't appear to be used.

Reply:
Oh, because it's hardcoded. Let's leave it for now then.

Reply:
We don't need them. As I remember, that balance check used to be sort of a sanity check ensuring we have enough balance on the accounts which we use to send transactions.